dist/circle-progress.js

```diff
@@ -369,21 +378,17 @@
     drawEmptyArc: function(v) {
       var ctx = this.ctx,
         r = this.radius,
-        t = this.getThickness(),
-        a = this.startAngle;
+        arcR = this.getArcRadius(),
```

Comment:
Yes, I left `this.radius` just to have easy access to `this.size / 2`, as I need that in order to calculate the actual radius and for the x, y coords for the circle.

I could either calculate the half coords everytime in `drawArc` and `drawEmptyArc`, as well as `getArcRadius`, or rename `this.radius` to something like `this.halfSize`, and use that instead of `this.radius`, just to avoid unnecessary math operations.

What do you think?

Reply:
This variable needs to be updated when `this.size` is changed. I'd rather drop it. There are only 2 places where it'd get replaced with `this.size / 2`.

Also, I still think it's reasonable to rename your `getArcRadius` method into `getRadius`.

I suggest the following naming:

```js
// ...
var c = this.size / 2, // center X/Y coords
  r = this.getArcRadius(),
  t = this.getThickness(),
  a = this.startAngle;
  // ...
  ctx.arc(c, c, r, a, a + Math.PI * 2 * v);
```